packages/stripe-graphql/README.md

```diff
@@ -0,0 +1,141 @@
+<h1 align="center">@nhost/stripe-graphql</h1>
+<h2 align="center">Stripe GraphQL</h2>
+
+<p align="center">
+  <img alt="npm" src="https://img.shields.io/npm/v/@nhost/stripe-graphql">
+  <img alt="npm" src="https://img.shields.io/npm/dm/@nhost/stripe-graphql">
+  <a href="LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-yellow.svg" alt="license: MIT" />
+  </a>
+</p>
+
+Stripe GraphQL API for Hasura Remote schemas.
+
+Connect data in your database with data from Stripe, via GraphQL.
+
+```graphql
+query {
+  users {
+    # User in your database
+    id
+    displayName
+    userData {
```

Comment:
Since we can't create columns on the `auth.users` table, we can't create a relationship to/from `users` since the remote schema requires a stripe customer id column.

It's only possible to create this stripe customer id column in another table (e.g. `userData`) and create the relationship at that table.